packages/sources/durin/package.json

```
@@ -0,0 +1,45 @@
+{
+  "name": "@chainlink/durin-adapter",
+  "version": "0.0.1",
```

We probably ought to update the example adapter, but I think we should start adapters on 1.0.0. @justinkaseman does this sound right?

```suggestion
  "version": "1.0.0",
```

---

My vote would be 1.0.0 as well